src/main/java/com/faforever/client/fa/relay/ice/IceAdapterImpl.java

```diff
@@ -198,15 +199,12 @@ public void onGpgGameMessage(GpgGameMessageEvent event) {
         ProcessBuilder processBuilder = new ProcessBuilder();
         processBuilder.directory(workDirectory.toFile());
         processBuilder.command(cmd);
-        processBuilder.environment().put("LOG_DIR", preferencesService.getFafLogDirectory().resolve("iceAdapterLogs").toAbsolutePath().toString());
+        processBuilder.environment().put("LOG_DIR", preferencesService.getIceAdapterLogDirectory().toAbsolutePath().toString());
```

Comment:
This is ignored by the ice adapter due to nobody fixing logback.

Reply:
So the path is hardcoded in the ice adapter?

Reply:
Anyway just leaving this in i guess?